programs/cli_wallet/main.cpp

```diff
@@ -257,6 +256,9 @@ int main( int argc, char** argv )
       if( !options.count( "daemon" ) )
       {
          auto wallet_cli = std::make_shared<fc::rpc::cli>( GRAPHENE_MAX_NESTED_OBJECTS );
+         
+         wallet_cli->set_regex_secret("\\s*(unlock|set_password)\\s*$");
```

Comment:
I think it's easier to understand if either have both `^` and `$`, or have none of them.

Reply:
done